services/graphite/parser.go

```diff
@@ -145,6 +147,19 @@ func (p *Parser) Parse(line string) (models.Point, error) {
 	return point, nil
 }
 
+// Apply extracts the template fields form the given line and returns the
+// measurement name and tags
+func (p *Parser) ApplyTemplate(line string) (string, map[string]string) {
```

Comment:
This needs to be external because I need to use it in Telegraf. Basically it is for anyone who wants to transform a statsd/graphite style buckets into influx style measurements and tags